src/hooks/useSelectRevision.ts

```diff
@@ -11,16 +10,14 @@ const useSelectRevision = () => {
   const dispatch = useAppDispatch();
   const { enqueueSnackbar } = useSnackbar();
 
-  const searchResults = useAppSelector(
-    (state: RootState) => state.search.searchResults,
-  );
+  const searchResults = useAppSelector((state) => state.search.searchResults);
 
   const checkedRevisions = useAppSelector(
-    (state: RootState) => state.checkedRevisions.revisions,
+    (state) => state.checkedRevisions.revisions,
   );
 
   const selectedRevisions = useAppSelector(
-    (state: RootState) => state.selectedRevisions.revisions,
```

Comment:
[question] should the type be added, will the linter complain?

Reply:
the typed hooks remove the need for specifying a type here: https://redux-toolkit.js.org/tutorials/typescript#define-typed-hooks